doc/release-notes/rl-2103.adoc

```diff
@@ -9,7 +9,28 @@ section is therefore not final.
 
 This release has the following notable changes:
 
-* Nothing has happened.
+* The `progrmas.broot.verbs` option is now a list rather than an
```

Comment:
Misspelled, also can be made into a link to the actual option documentation using
```suggestion
* The <<opt-programs.broot.verbs>>` option is now a list rather than an
```